### ISSUE TITLE
.svg files are being placed in wrong location

### ISSUE THREAD

**Issue by tatemz** [agentk/gulp-fontgen#16]
Perhaps this is a problem with `fontgen` itself, but all of the .svg files are being generated and placed into the directory from which gulp is running (or rather from where gulp has called node).

I have properly configured my task with a valid `dest` option:
```
gulp.task('fontgen', function() {
  return gulp.src('app/fonts/**/*.{ttf,otf}')
    .pipe(fontgen({
      dest: "./dist/fonts/",
      css_fontpath: '../fonts/',
      embed: ['svg'],
    }));
});
```

**Comment by sgen** [comment]
Technically we never officially supported embedding svg files. So Im changing the label to enhancement.

**Comment by sgen** [comment]
Feature added in agentk/fontfacegen@d9e87e6a2531705e09c940575620851a6d551514

**Comment by tatemz** [comment]
Eh... I think this is still a bug. I removed the `embed: ['svg']` and the issue still exists...

Here is how my project was compiled after my gulp run.

```
├── app
│   └── fonts
│       └── fontfile.ttf
├── dist
│   └── fonts
│       ├── fontfile.css
│       ├── fontfile.eot
│       ├── fontfile.svg
│       ├── fontfile.ttf
│       └── fontfile.woff
├── fontfile.svg <-- This is a clone and shouldn't be here
└── gulpfile.js
```

**Comment by sgen** [comment]
I cannot reproduce this. What are your node, and gulp-fontgen versions?

**Comment by tatemz** [comment]
```
gulp-fontgen: ^0.1.7
```

```
npm: '2.14.12',
node: '4.2.6',
```

**Comment by sgen** [comment]
Try updating to gulp-fontgen 0.1.8 and see if the issue persists.

**Comment by tatemz** [comment]
No luck... problem persists. 😢

**Comment by sgen** [comment]
Which operating system are you using?

**Comment by tatemz** [comment]
Docker for Mac (beta) using the `node:4.2` docker image

I suppose it might be either Docker or the fact that I'm using 4.2...

**Comment by sgen** [comment]
If your on Mac this might be related to agentk/fontfacegen#24

Ive just pushed the fix for that to npm. Try updating gulp-fontgen to 0.1.9 and see if the issue persists.

**Comment by tatemz** [comment]
No luck still... I'll see if I can find some time this evening to dive in and see what more debugging I can do.

**Comment by sgen** [comment]
Alright. Im not sure how this is happening.

The target destination for the .svg is derived from the source file name and target destination provided in the configuration [lib/fontfacegen.js](https://github.com/agentk/fontfacegen/blob/master/lib/fontfacegen.js#L16), [lib/configure.js](https://github.com/agentk/fontfacegen/blob/master/lib/configure.js#L16-L18).

And this is passed to tt2svg / batik-ttf2svg (https://github.com/agentk/fontfacegen/blob/master/lib/ttf2svg.js#L10).

the embedding is handled in [lib/stylesheets.js](https://github.com/agentk/fontfacegen/blob/master/lib/stylesheets.js#L96) and svg encoding in encoding	[lib/encode.js](https://github.com/agentk/fontfacegen/blob/master/lib/encode.js#L22).

drop some breakpoints / console.log()s in and see whats going on.

**Comment by tatemz** [comment]
Gotcha. One thing to note is that it is duping the `.svg` files and placing one in the correct `dest` location and one where ever the `gulpfile.js` is.

**Comment by sgen** [comment]
@tatemz Did you ever take a look at this?

**Comment by sgen** [comment]
Fixed by agentk/fontfacegen#36
